db/migrate/20190424143344_add_type_to_transient_registrations.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddTypeToTransientRegistrations < ActiveRecord::Migration
+  def change
+    add_column :transient_registrations, :type, :string
```

Comment:
Instead of creating a different migration, what about:
```
def up 
  add_column :transient_registrations, :type, :string
  execute <<-SQL
     UPDATE transient_registrations 
     SET type = 'TransientRegistration';
  SQL
end

def down
  remove_column ....
end
```

Reply:
Implemented! https://github.com/DEFRA/waste-exemptions-engine/pull/138/commits/946e1f81cef9b7e93fd1463503a3321a044e2025